package-lock.json

```diff
@@ -52,7 +52,7 @@
         "date-fns-tz": "^2.0.0",
         "dom-serializer": "^0.2.2",
         "domhandler": "^4.3.0",
-        "expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#77d0b150ba6bfbe7a64b3c3e30b65592b2e58c4a",
+	"expensify-common": "git+ssh://git@github.com/Expensify/expensify-common.git#a8ed0f8e1be3a1e09016e07a74cfd13c85bbc167",
```

Comment:
I noticed diff here in package-lock.json. Can you please fix this indent?

Reply:
Fixed!